### PR TITLE
Review taxonomic profiling

### PR DESCRIPTION
**Pull request by paulzierep**
* fixed wrong image 
* small changes in wording 
* small addition to content
* added info on difference to amplicon analysis
* added citation

Additional note:

Should the MetaPhlAn part be in the Details ? I think it would be better in the main text.

### CODE REVIEW

File: topics/metagenomics/tutorials/taxonomic-profiling/tutorial.md [hunk 1]
```diff
@@ -35,27 +36,31 @@ The term **"microbiome"** describes "a characteristic microbial community occupy
...
-2. **DNA-to-Protein** : compare sequencing reads with protein databases (more computationally intensive because of analysis of all six frames of potential DNA-to amino acid translation, with tools like DIAMOND
-3. **Marker based**: search for marker genes (e.g. 16S rRNA sequence) in reads, which is quick, but introduces bias, with tools like MetaPhlAn ({% cite blanco2023extending %})
+1. **DNA-to-DNA**: comparison of sequencing reads with genomic databases of DNA sequences with tools like Kraken ({% cite Wood2014 %})
+2. **DNA-to-Protein** : comparison of sequencing reads with protein databases (more computationally intensive because all six frames of potential DNA-to amino acid translations need to be analyzed) with tools like DIAMOND)
+3. **Marker based**: searching for marker genes (e.g. 16S rRNA sequence) in reads, which is quick, but introduces bias, with tools like MetaPhlAn ({% cite blanco2023extending %})
+4. **Mixed**: Combined search using e.g. maker genes and translated searches, with tools like HUMAnN ({% cite franzosa_species-level_2018 %})
```
Comment (bebatut) [inline topics/metagenomics/tutorials/taxonomic-profiling/tutorial.md:63]: Not sure about that. HUMAnN does not do taxonomic profiling, it relies completely on MetaPhlAn for that

Comment (paulzierep) [reply topics/metagenomics/tutorials/taxonomic-profiling/tutorial.md:63]: Yes, that was a brain fart from me, it does indeed use MetaPhlAn and diamond for translated search, but does not traxonomic profile the found proteins ... but when I think of an example of **DNA-to-Protein** for taxonomy, can diamond also assing taxonomy ? Whichever, I remove that part.

**Comment by bebatut** [comment]
I put metaphlan in the details because the results are not really good with this dataset. Do you think it should still be in the main text?

**Comment by bebatut** [comment]
I will merge this PR so I can record the video for Smorgsboard. We can update the tutorial later with maybe new dataset where we get better results with MetaPhlAn

**Review by bebatut — APPROVED** [review]

**Comment by paulzierep** [comment]
all good by me !! I was just wondering about the metaphlan part in a block, but the reason is good !! Great work !

**Pull request merged**